### PR TITLE
ci: support `prefer-template`

### PR DESCRIPTION
**Pull request by splincode**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Code style update
- [x] Build or CI related changes
- [ ] Documentation content changes

### CODE REVIEW

**Review by nsbarsukov — APPROVED** [review]

File: projects/addon-doc/src/utils/color-conversion.ts [hunk 1]
```diff
@@ -1,14 +1,11 @@
...
-                const hex = x.toString(16);
+    return `#${[r, g, b]
+        .map(x => {
+            const hex = x.toString(16);
 
-                return hex.length === 1 ? '0' + hex : hex;
-            })
-            .join('')
-    );
+            return hex.length === 1 ? `0${hex}` : hex;
```
Comment (waterplea) [inline projects/addon-doc/src/utils/color-conversion.ts:6]: Do you think we can use `padStart` here? It seems to be supported by Safari 10

File: projects/addon-editor/utils/rgb-to-hex.ts [hunk 2]
```diff
@@ -1,6 +1,6 @@
 export function rgbToHex(r: number, g: number, b: number): string {
     return [r, g, b]
         .map(x => x.toString(16))
-        .map(hex => (hex.length === 1 ? '0' + hex : hex))
+        .map(hex => (hex.length === 1 ? `0${hex}` : hex))
```
Comment (waterplea) [inline projects/addon-editor/utils/rgb-to-hex.ts:4]: Same here

File: projects/demo/src/modules/app/customization/customization.component.ts [hunk 3]
```diff
@@ -81,7 +81,7 @@ export class TuiCustomizationComponent implements AfterViewInit {
     onModelChange(variable: string, value: string | number) {
         this.variables = {
             ...this.variables,
-            [variable]: typeof value === 'string' ? value : value + 'px',
+            [variable]: typeof value === 'string' ? value : `${value}px`,
```
Comment (waterplea) [inline projects/demo/src/modules/app/customization/customization.component.ts:84]: We have a utility function `px` for that.

File: projects/kit/components/text-area/test/text-area.component.spec.ts [hunk 4]
```diff
@@ -134,7 +134,7 @@ describe('TextArea', () => {
 
             const maxHeight = component.rows * LINE_HEIGHT_L;
 
-            expect(getScrollbar().style.maxHeight).toEqual(maxHeight + 'px');
+            expect(getScrollbar().style.maxHeight).toEqual(`${maxHeight}px`);
```
Comment (waterplea) [inline projects/kit/components/text-area/test/text-area.component.spec.ts:137]: Same here

File: projects/kit/components/text-area/test/text-area.component.spec.ts [hunk 5]
```diff
@@ -144,7 +144,7 @@ describe('TextArea', () => {
...
```
Comment (waterplea) [inline projects/kit/components/text-area/test/text-area.component.spec.ts:147]: And here

File: projects/kit/components/text-area/test/text-area.component.spec.ts [hunk 6]
```diff
@@ -154,7 +154,7 @@ describe('TextArea', () => {
...
```
Comment (waterplea) [inline projects/kit/components/text-area/test/text-area.component.spec.ts:157]: And here

**Review by waterplea — APPROVED** [review]

File: projects/addon-doc/src/utils/color-conversion.ts (hunk 1 above)
Comment (splincode) [reply projects/addon-doc/src/utils/color-conversion.ts]: fixed

File: projects/addon-editor/utils/rgb-to-hex.ts (hunk 2 above)
Comment (splincode) [reply projects/addon-editor/utils/rgb-to-hex.ts]: fixed

File: projects/demo/src/modules/app/customization/customization.component.ts (hunk 3 above)
Comment (splincode) [reply projects/demo/src/modules/app/customization/customization.component.ts]: fixed

File: projects/kit/components/text-area/test/text-area.component.spec.ts (hunk 4 above)
Comment (splincode) [reply projects/kit/components/text-area/test/text-area.component.spec.ts]: fixed

File: projects/kit/components/text-area/test/text-area.component.spec.ts (hunk 5 above)
Comment (splincode) [reply projects/kit/components/text-area/test/text-area.component.spec.ts]: fixed

File: projects/kit/components/text-area/test/text-area.component.spec.ts (hunk 6 above)
Comment (splincode) [reply projects/kit/components/text-area/test/text-area.component.spec.ts]: fixed

**Pull request merged**